python/test/openmldb_client_test.py

```diff
@@ -314,10 +316,48 @@ def test_parameterized_query(self):
       (1008, '2021-01-02', 'province3', 'city9', 9, 1590738998000),
       ]
     self.check_result(rs, expectRows)
- 
 
+# test sqlalchemy Table-object-based API in pytest style
+class TestSqlalchemyAPI:
+
+    def setup_class(self):
+        self.engine = db.create_engine('openmldb:///db_test?zk=127.0.0.1:6181&zkPath=/onebox')
```

Comment:
The engine is not used in other methods. We could make it as local variable instead of object member.

Reply:
engine is used in `metadata.create(self.engine)`